### PR TITLE
Converted OracleFeed to use Overflow table. 

### LINKED ISSUE #261: Convert `/prices/[symbol].tsx` Oracles table used to `<OverflowTable/>`
#### What would you like to be added:

Currently, in /prices/* the Oracle table was implemented before OverflowTable was implemented https://github.com/DeFiCh/scan/blob/main/src/components/prices/PriceOracleTable.tsx we need to switch to use OverflowTable for better maintainability of the code

/triage accepted
/assign @siradji 
/area ui-ux

### PR DESCRIPTION
**Pull request by siradji**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:

Implemented Overflow table to make oraclesFeed more responsive
 
#### Which issue(s) does this PR fixes?:

Fixes #261 

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on multiple web browsers
- [x] Tested responsiveness (e.g, iPhone, iPad, Desktop)
- [x] No console errors
- [x] Unit tests*
- [x] Added e2e tests*

### CODE REVIEW

File: cypress/integration/pages/prices/prices-[symbol].spec.ts [hunk 1]
```diff
@@ -1,59 +1,57 @@
...
-  })
 
-  it('should have section: Pricing feeds by', () => {
-    cy.get('main section')
-      .should('contain.text', "Pricing feeds by")
-  })
+    it('should have section: Pricing feeds by', () => {
+        cy.get('main section')
+            .should('contain.text', "Pricing feeds by")
+    })
```
Comment (fuxingloh) [inline cypress/integration/pages/prices/prices-[symbol].spec.ts:56]: indentation is off

Comment (siradji) [reply cypress/integration/pages/prices/prices-[symbol].spec.ts:56]: I ran the formatter and nothing is changing.

Comment (fuxingloh) [reply cypress/integration/pages/prices/prices-[symbol].spec.ts:56]: https://github.com/DeFiCh/scan/pull/273 will fix cypress indentation rules

Comment (fuxingloh) [reply cypress/integration/pages/prices/prices-[symbol].spec.ts:56]: rebase this PR from main with #273 changes to fix indentation issues

Comment (siradji) [reply cypress/integration/pages/prices/prices-[symbol].spec.ts:50]: fixed @fuxingloh .

**Comment by fuxingloh** [comment]
please rebase properly

**Review by joeldavidw — APPROVED** [review]

**Pull request merged**